src/tasks.js

```diff
@@ -34,6 +35,11 @@ function createTask({
     task.isOngoing = true;
   }
 
+  if (isMilestone) {
+    task.isOngoing = false; // Override isOngoing for milestones
+    task.isMilestone = true;
+  }
```

Comment:
NIT Milestones probably shouldn't have a `length` attribute either.

Reply:
True, but maybe this could be fixed more generally in a tasks parsing step. I don't love overriding the data here as it is?